NASA-AMMOS/aerie-ui
# Profile fetching failure toast

Show toast on profile and external profile fetching failure. Closes #1105.

Testing:
1. Load a plan
2. Temporarily modify the GQL `GET_PROFILES` and `GET_PROFILES_EXTERNAL` to be invalid.
3. Refresh the plan
4. Toasts should appear alerting you that profile fetching failed
